### PR TITLE
feat(AV-1528): Resource page implementation styling

### PR DESCRIPTION
**Pull request by teravill**
Contains resource page styling changes, mainly new additions to the resource.less file.

### CODE REVIEW

File: src/less/components/resource.less [hunk 1]
```diff
@@ -1,16 +1,95 @@
...
+
+  .module-small-title{
+    height: 24px;
+    color: rgb(95, 104, 109);
+    font-size: 16px;
+    font-family: @font-family-sans-serif; 
+    font-weight: normal;
+    line-height: 24px;
+    padding-top: 10px;
+  }
```
Comment (beazil) [inline src/less/components/resource.less]: I'd recommend to use variables for font's at least (found from `variables/fonts.less`). `@font-size-slightly-smaller-base` and `@line-height-base` should be fine in here. Also I would avoid hard coded height for elements if there's no good reason for that (like horizontal scroll etc.)

Also most of colors has variable in `variables/palette.less` and those should be used (if there isn't suitable one, maybe one should be added).

File: src/less/components/resource.less [hunk 2]
```diff
@@ -1,16 +1,95 @@
...
+  a{
+    color: rgb(42, 110, 187);
+    font-family: @font-family-sans-serif;
+    font-size: 14px;
+    font-weight: normal;
+    height: 18px;
+    letter-spacing: 0.4px;
+    line-height: 18px;
+    text-transform: uppercase;
+    width: 96px;
```
Comment (beazil) [inline src/less/components/resource.less]: font-size/line-height- and color- variables could be used for some of these

File: src/less/components/resource.less [hunk 3]
```diff
@@ -1,16 +1,95 @@
...
+.toolbar-resource-secondary{
+  max-width: 1110px;
+  margin: 0 auto;
+  padding: 0 20px;
+
+  i{
+    border-radius: 0px;
+    height: 16px;
+    width: 16px;
+    color: #2a6ebb;
```
Comment (beazil) [inline src/less/components/resource.less]: Color variable from palette could be used

**Review by beazil — CHANGES_REQUESTED** [review]
There's multiple places where font-sizes/line-heights/colors could be replaced with less-variables. It is good practice at least for fonts and colors, as if they're changed at some point, we most probably want to do that everywhere in application code, and instead of find-replace we could just change value for variable. Also, hard coded element sizing should be avoided if there isn't specific reason for that

**Comment by teravill** [comment]
Implemented the suggested changes

File: src/less/components/resource.less (hunk 1 above)
Comment (teravill) [reply src/less/components/resource.less]: Resolved in [4bd529b](https://github.com/vrk-kpa/opendata-assets/pull/38/commits/4bd529b924677e0b078b615982ed2ec2625bf2c1)

File: src/less/components/resource.less (hunk 2 above)
Comment (teravill) [reply src/less/components/resource.less]: Resolved in [4bd529b](https://github.com/vrk-kpa/opendata-assets/pull/38/commits/4bd529b924677e0b078b615982ed2ec2625bf2c1)

File: src/less/components/resource.less (hunk 3 above)
Comment (teravill) [reply src/less/components/resource.less]: Resolved in [4bd529b](https://github.com/vrk-kpa/opendata-assets/pull/38/commits/4bd529b924677e0b078b615982ed2ec2625bf2c1)

**Review by beazil — APPROVED** [review]

**Pull request merged**